builder/remotecontext/detect.go

```diff
@@ -33,12 +33,7 @@ func Detect(config backend.BuildConfig) (remote builder.Source, dockerfile *pars
 	case remoteURL == "":
 		remote, dockerfile, err = newArchiveRemote(config.Source, dockerfilePath)
 	case remoteURL == ClientSessionRemote:
-		res, err := parser.Parse(config.Source)
```

Comment:
FWIW; this function (`remotecontext.Detect()`) was only used by the classic builder, and not used in the CLI, so I thought this would be a good spot to produce the error (initially had it in the old place, but thought this would be cleaner)